webCommander/centralHub/views.py

```diff
@@ -1,7 +1,29 @@
 from django.shortcuts import render,get_object_or_404
-from centralHub.models import TwitterUser
+from .forms import SubmitXUser
+from centralHub.models import TwitterUser, TwitterUserPosts
+from .xTwitter.twitterAPI import x_userlookup
 
 # Create your views here.
+
+def submit_x_user(request):
+    if request.method == 'POST':
+        form_handle = SubmitXUser(request.POST)
+        if form_handle.is_valid():
+            entered_handle = form_handle.cleaned_data['handle']
+            x_api_response = x_userlookup(entered_handle)
```

Comment:
I thought is_valid was a property of forms. Will probably need some help on this.

Reply:
Using is_valid as a property (without parentheses) implies it should be a quick, low-cost operation. 

However, in Django, is_valid() is a method (with parentheses) because it may involve running multiple validation checks, some of which could be more computationally intensive than others. 

For more computationally intensive work, it's better to use a method. This is likely why they chose to implement it as a method here.

Happy to show mentioned refactoring on our next call ...